### PR TITLE
Remove one additional access to google stuff in shaded curator package.

### PR DESCRIPTION
**Pull request by fapifta**
## What changes were proposed in this pull request?

I found one more use of curator, it seems that in one of the tests, we are using something shaded into it.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3353

## How was this patch tested?
local build.

### CODE REVIEW

File: hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSetDiskChecks.java [hunk 1]
```diff
@@ -18,36 +18,36 @@
...
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
-
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Iterables;
+import org.apache.commons.io.FileUtils;
```
Comment (adoroszlai) [inline hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSetDiskChecks.java:23]: Can we avoid moving imports back and forth? :)

**Review by adoroszlai — CHANGES_REQUESTED** [review]
Nice catch @fapifta for the import from shaded package.

**Comment by adoroszlai** [comment]
This is obsoleted by apache/hadoop-ozone#818.

**Comment by fapifta** [comment]
Closing this one.

**Pull request closed without merging**